Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM node:6 AS builder
+
+ENV ENV_NODE=production
```

Comment:
Convention is to use `NODE_ENV`, many libraries use this convention to check environment variable for production use case. So better to rename it.